### DISCUSSION TITLE
So quiet around here!

### DISCUSSION THREAD

**Discussion by CCS86** [supermerill/SuperSlicer#2891] [General]
Everything good @supermerill? It has been over a month since the last action.  :)

**Comment by Seppideluxe** [comment]
Does anyone in the community have direct contact and can find out if everyone is ok? Of course, always with the necessary respect for privacy

**Reply by ntcong** [reply]
He had covid last month, probably taking some time off to recover

**Reply by Seppideluxe** [reply]
Thanks for the info and most of all all the best for him!!! Hopefully without late effects.

**Comment by CCS86** [comment]
He's back!

**Comment by Seppideluxe** [comment]
then everything good and above all not lasting

**Comment by moo00se** [comment]
::cricket noises::

**Comment by Ojref** [comment]
I too have been concerned, SS seems like its fell behind Prusaslicer and trailing it for updates

**Comment by CCS86** [comment]
It has been another quiet month

**Comment by Newcal** [comment]
Is supermerill alright ??

**Comment by Ojref** [comment]
I'm at the point of moving on, I don't know what the issue is, I hope all is well and due respect of course if not. Cheers and good luck all.

**Comment by jhlchu** [comment]
Anyone know of a good fork of SS or PS (other than SS)?

**Comment by moo00se** [comment]
UPDATE: Merill seems to be back as of 10/26. Trying to get back to work. Seen in the Discord Server.

Happy that you're ok Merill! Take your time getting back.

**Reply by Newcal** [reply]
Awwwww !! Excellent, I am glad he's ok !

Have a good one everybody :)

**Comment by CCS86** [comment]
Still no actions in the SS repo...

**Reply by Newcal** [reply]
Hi mate.

Even though we all love new updates with new features, if we haven't heard from him, he may have a good reason.. So let's wait and maybe some of us can help him developing the software in the meantime ?

Cheers

**Reply by jhlchu** [reply]
There is no developing, he was the sole maintainer. Can't merge PRs without his approval, can't make builds from actions without him triggering it.
Only thing to do is to make your own fork.

**Reply by Newcal** [reply]
Sounds like something which could be done anyway, isn't it ?

**Reply by jhlchu** [reply]
Well SS has been lagging too far behind PS. It's probably better to just fork the main PS and add the SS functions that you want instead of forking SS.

**Reply by Newcal** [reply]
Well, can anyone help then ?

**Reply by jhstatewide** [reply]
@Newcal What do you need help with?

**Reply by usulrasolas** [reply]
@jhstatewide I think the 2.5 merge, all the main branch features but ss granularity

**Reply by jhstatewide** [reply]
What do you have working so far?

**Reply by Newcal** [reply]
> @Newcal What do you need help with?

Sorry, I genuinely don’t get your question.

I am not asking for help. I was saying, if we could help him with SS, what could we do.

**Reply by jhstatewide** [reply]
Send him some $$$

**Reply by CCS86** [reply]
I have been sponsoring. But, it's pretty hard to justify continuing when no development is occurring.

**Comment by CCS86** [comment]
:(

![image](https://user-images.githubusercontent.com/12439543/202832166-0a6a70af-0b38-45d8-b7fb-e0ad9cb88030.png)

**Reply by Newcal** [reply]
well...

**Comment by rohrbrecher** [comment]
For the time being: Is there a description how to use PrusaSlicer with Klipper/Mainsail? I tried some months ago but I quickly realized that sending files to Klipper automatically doesn't seem to work and also the generated gcode files seemed to give me some errors. So are there some basic settings everyone must use when using PS and Klipper?

**Reply by EchoAlpha519** [reply]
The Klipper website has some advice on how to set up your slicer. https://www.klipper3d.org/Slicers.html

**Reply by Nossolov** [reply]
3 printers with klipper/mainsail on PC and Mac under debian, and raspberry for me, and no problems. Once with the use of ArcWelder.
Nothing spécial with settings, just adjustment to printers. Gcode generated with SS2.3, SS2.4, SS2.5 and prusa 2.5.

**Comment by Stooovie** [comment]
It's getting stale but tbh Prusa Slicer had its last update in Sep 7

**Reply by Garryb123** [reply]
Does anyone know the size of the PrusaSlicer developer team? I'm assuming they are full-time salaried programmers. It's amazing Merill and those helping him have achieved all they have as a side project.

**Reply by doctor-bob** [reply]
but what is going into PS from SS...? I still dont like the prusa, but with thing going nowhere in SS, I am asking myself if theres a connection between the ignorance of the PS-team and the bugged-to-the -point-of-being-useless state of the newest SS-release.....it smells fishy to me....as if somebody is ruining SS intentinoaally.

**Reply by jhstatewide** [reply]
@doctor-bob, I totally get where you're coming from with your concerns about the recent issues with the latest release of superslicer. It's definitely frustrating to see problems with the software and to wonder if there's some shady stuff going on behind the scenes. But it's important to remember that software development is a tricky business and bugs and issues are just part of the process.

Also, it's worth noting that the superslicer and prusa slicer teams are separate entities, and the prusa team probably isn't directly involved in superslicer's development. open-source software development is often done by volunteers who are working on the project in their free time, so lack of resources or management problems can cause issues.

The open-source community is all about collaboration and transparency, so I'm sure the superslicer team is working to fix any issues as soon as they're made aware of them. In the meantime, you might want to check the superslicer discord or GitHub page for updates and progress on the software, as well as any possible workarounds for any bugs you might be experiencing.

In summary, while it's totally understandable to be frustrated, let's not jump to conclusions without more information. let's give the SS team the chance to investigate and address the problem and be patient in the meantime.

**Reply by Nossolov** [reply]
Sorry, but in french ...
SS est un super soft, surement le meilleur des slicers non Arachne. N'oublions pas que Cura est devenu payant en partie, que Simplify3D n'a jamais eu de version 5, que Prusa ne veut pas intégrer la correction des trous verticaux. L'integration d'arachne demande sans doute beaucoup de travail (travail que je suis bien incapable de faire), et comme le dit jhstatewide, le(s) développeur(s) de SS n'ont pas que ça à faire. C'est déjà admirable de pouvoir profiter de leur travail aussi facilement et librement. Alors, oui, il faut attendre. Et quand la version arachne sera disponible, ce sera un nouveau cadeau de plus à la communauté.  Merci à eux.

**Reply by numanair** [reply]
@Nossolov SS does have an Arachne release already.

**Reply by Nossolov** [reply]
Yes you are right : 2.5.59.0
I try this. Quality of the print is not as good as last V2.4 with my printers. Prusa 2.5.0 is better when you really need arachne, and it is not so often for me. But Prusa don't support vertical hole correction, and it's very important when you have several printers. Cura support arachne, vertical hole, do the job, but it's "une usine à gaz". So, SS 2.4 is still the better for me.

**Reply by Visthum** [reply]
Arachne in SS is unusable, gives you lots of gaps. Hole correction will be included in prusa 2.6 which is releasing very soon, probably next week. You can measure stuff and correct hole dimension etc.

**Reply by Nossolov** [reply]
Just download and try prusa2.6alpha2. Still no hole correction, or i can't find it.

**Comment by CCS86** [comment]
Woah!

![image](https://user-images.githubusercontent.com/12439543/211179067-b1c7cccf-f027-437c-81ba-a74545fc6614.png)

**Reply by CCS86** [reply]
Man, I was hoping the progress would continue.

**Comment by ModMike** [comment]


**Reply by jhstatewide** [reply]
https://github.com/supermerill/SuperSlicer/issues/2989
